src/SerilogWeb.Classic.Mvc/Classic/Mvc/SerilogWebClassicHandleErrorAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Web.Mvc;
+using SerilogWeb.Classic.Extensions;
+
+namespace SerilogWeb.Classic.Mvc
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true)]
+    internal class SerilogWebClassicHandleErrorAttribute : HandleErrorAttribute
```

Comment:
`HandleErrorAttribute` comes with some behavior defined already, and for instance may trigger displaying a view, or interacting with the `Custom Errors` section of `web.config` , so I'm afraid that by introducing this in the calling app, we may introduce undesired behavior.

Any chance we can use `IExceptionFilter` instead (i.e. just an interface, so no behavior other than the one we choose to add) ? 

WDYT ? 

see https://docs.microsoft.com/es-es/dotnet/api/system.web.mvc.iexceptionfilter?view=aspnet-mvc-5.2

and the source code for `HandleErrorAttribute` : https://github.com/aspnet/AspNetWebStack/blob/749384689e027a2fcd29eb79a9137b94cea611a8/src/System.Web.Mvc/HandleErrorAttribute.cs

Reply:
Good one! I borrowed the idea from Elmah.Mvc. They use a `HandleErrorAttribute`. I'll try `IExceptionFilter`.

Reply:
Needed some minor changes, but got it to work with only `IExceptionFilter`